netci.groovy

```diff
@@ -797,7 +797,9 @@ combinedScenarios.each { scenario ->
                                 case 'arm64':
                                     assert scenario == 'default'
                                     buildCommands += "build.cmd ${lowerConfiguration} ${architecture} skiptestbuild /toolset_dir C:\\ats"
-                                    // Add archival.  No xunit results for x64 windows
+                                    buildCommands += "C:\\arm64PostBuild.cmd ${WORKSPACE} ${architecture} ${lowerConfiguration}"
```

Comment:
Right, when you have incoming parameters to a job or default variables that are set, they get set in the environment.  So depending on what kind of shell you're working with (bash/cmd) you'd address the variables in the appropriate way.